enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1391,6 +1432,16 @@ func (ar *actionRunner) workspaceStatusEvent() *bespb.BuildEvent {
 	}
 }
 
+// Returns the bazel command - i.e. `build` or `run`
+func getBazelCommand(bazelArgs []string) string {
+	for _, arg := range bazelArgs {
+		if !strings.HasPrefix(arg, "--") {
+			return arg
+		}
+	}
+	return ""
+}
```

Comment:
I think ideally we'd centralize any parsing logic in the CLI since it has more information about the bazel flag schema (`cli/parser` package). Could the CLI maybe detect whether we're doing a remote run and then pass --script_path as an additional arg?

Reply:
Good idea - done